src/card-validator/card-validator.ts

```diff
@@ -70,4 +72,22 @@ valid.creditCardType.addCard({
   },
 })
 
-export default valid
+function maskCardNumber(type: string) {
```

Comment:
acha que seria interessante ter um parâmetro com o valor do char?

Ex:

```suggestion
function maskCardNumber(type: string, character = '0') {
```